packages/statemanager/CHANGELOG.md

```diff
@@ -6,6 +6,30 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 (modification: no type change headlines) and this project adheres to
 [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 2.1.0 - 2023-10-23
+
+### New Diff-Based Code Cache
+
+This release introduces a new code cache implementation, see PR [#3022](https://github.com/ethereumjs/ethereumjs-monorepo/pull/3022) and [#3080](https://github.com/ethereumjs/ethereumjs-monorepo/pull/3080). The new cache complements the expanded account and storage caches and now also tracks stored/deployed-code-changes along commits and reverts and so only keeps code in the cache which made it to the final state change.
+
+The new cache is substantially more robust towards various type of revert-based attacks and grows a more-used cache over time.
+
+### Peformance Option to store Storage Keys with Prefix
+
+This release introduces a new option `prefixStorageTrieKeys` which triggers the underlying trie to store storage key values with a prefix based on the account address, see PR [#3023](https://github.com/ethereumjs/ethereumjs-monorepo/pull/3023). This significantly increases performance for consecutive storage accesses for the same account on especially larger tries, since trie node accesses get noticeably faster to be performed by the underlying key-value store since values are stored close to each other.
```

Comment:
```suggestion
This release introduces a new option `prefixStorageTrieKeys` which triggers the underlying trie to store storage key values with a prefix based on the account address, see PR [#3023](https://github.com/ethereumjs/ethereumjs-monorepo/pull/3023). This significantly increases performance for consecutive storage accesses for the same account on especially larger tries, since trie node accesses get noticeably faster when performed by the underlying key-value store since values are stored close to each other.
```